src/pages/api/xml.ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { convertToXML } from '../../lib/xml/xmlConverter'
+import { setDate } from '../../lib/utils'
+import { calculate } from '../../lib/calculation'
+import { TaxForm } from '../../types/TaxForm'
+import { TaxFormUserInput } from '../../types/TaxFormUserInput'
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<void> => {
+  if (!req.body.taxFormUserInput) {
+    return res.status(500).json({ error: 'invalid data' })
+  }
+
+  const formInput: TaxFormUserInput = JSON.parse(req.body.taxFormUserInput)
+  const form: TaxForm = calculate(setDate(formInput))
```

Comment:
```suggestion
  const form: TaxForm = calculate(setDate(formInput, new Date()))
```

Tu musi byt toto, lebo `setDate` je zla neintuitivna funkcia a automaticky tam dosadi nejaky datum v minulosti :D Myslim ze som to tak spravil kvoli testom, ale chcelo by to refaktornut

Reply:
Nemali by sme potom to opravit aj tu?

https://github.com/priznanie-digital/priznanie-digital/blob/master/src/pages/_app.tsx#L44

Reply:
Hej, je to bugnute. https://github.com/priznanie-digital/priznanie-digital/pull/162 tento PR by to mal riesit vsade, takze mozes to nechat tak ako to je teraz

Reply:
Sorry ak si to uz fixol tiez :(

Reply:
nevadi revertnem to :)